packages/client/components/TeamPrompt/TeamPromptWorkDrawer.tsx

```diff
@@ -0,0 +1,102 @@
+import {Close} from '@mui/icons-material'
+import graphql from 'babel-plugin-relay/macro'
+import React, {useState} from 'react'
+import {PreloadedQuery, usePreloadedQuery} from 'react-relay'
+import {
+  TaskStatusEnum,
+  TeamPromptWorkDrawerQuery
+} from '../../__generated__/TeamPromptWorkDrawerQuery.graphql'
+import NullableTask from '../NullableTask/NullableTask'
+import {TaskStatus} from '../../types/constEnums'
+import clsx from 'clsx'
+import {meetingColumnArray} from '../../utils/constants'
+import {taskStatusLabels} from '../../utils/taskStatus'
+import halloweenRetrospectiveTemplate from '../../../../static/images/illustrations/halloweenRetrospectiveTemplate.png'
+
+interface Props {
+  queryRef: PreloadedQuery<TeamPromptWorkDrawerQuery>
+  onToggleDrawer: () => void
+}
+
+const TeamPromptWorkDrawer = (props: Props) => {
+  const {queryRef, onToggleDrawer} = props
+  const data = usePreloadedQuery<TeamPromptWorkDrawerQuery>(
+    graphql`
+      query TeamPromptWorkDrawerQuery($after: DateTime, $userIds: [ID!]) {
+        viewer {
+          tasks(first: 1000, after: $after, userIds: $userIds)
+            @connection(key: "UserColumnsContainer_tasks", filters: ["userIds"]) {
+            edges {
+              node {
+                ...NullableTask_task
+                id
+                status
+              }
+            }
+          }
+        }
+      }
+    `,
+    queryRef
+  )
+  const {viewer} = data
+  const {tasks} = viewer
+
+  const [selectedStatus, setSelectedStatus] = useState<TaskStatusEnum>(TaskStatus.DONE)
+  const selectedTasks = tasks.edges
+    .map((edge) => edge.node)
+    .filter((task) => task.status === selectedStatus)
+
+  return (
+    <>
+      <div className='px-4 pt-4'>
+        <div className='flex justify-between pr-4'>
+          <div className='text-base font-semibold'>Your Tasks</div>
+          <div className='cursor-pointer text-slate-600 hover:opacity-50' onClick={onToggleDrawer}>
+            <Close />
```

Comment:
-1 Can we make the X in the same location for both drawers? That will minimize the visual change when clicking between them.

<img width="373" alt="Screenshot 2023-08-22 at 11 51 42 AM" src="https://github.com/ParabolInc/parabol/assets/5514175/d15f52ab-c084-42ab-907d-1dbcfec59540">
<img width="383" alt="Screenshot 2023-08-22 at 11 51 50 AM" src="https://github.com/ParabolInc/parabol/assets/5514175/8a2808de-7f97-4716-9e82-21e259678f02">

Reply:
Fixed!